doc/concepts/replication/supervised_failover.rst

```diff
@@ -0,0 +1,182 @@
+    In this case, one coordinator is active and others are passive.
+
+When a cluster and failover coordinators are up and running, a failover coordinator :ref:`appoints one instance to be a master <supervised_failover_overview_appoint_master>` if there is no master instance in a replica set.
+Then, the following events may occur:
+
+-   If a master instance fails, a failover coordinator performs an :ref:`automated failover <supervised_failover_overview_perform_failover>`.
+-   If an active failover coordinator fails, another coordinator becomes active and performs read-write appointments.
+
+
+.. _supervised_failover_overview_appoint_master:
+
+Appointing a master instance
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A failover coordinator uses so-called appointments to switch an instance to read-write mode.
+For each appointment, there is a read-write deadline, which is renewed periodically to prevent split-brain.
+
+A failover coordinator appoints a master instance as follows:
+
+-   If a replica set has no read-write instance, a failover coordinator appoints one to be a master instance.
+-   If a replica set has one read-write instance and there is no previous appointment, a failover coordinator creates an appointment for this instance and sets a read-write deadline.
+-   If a replica set has one read-write instance and there is a previous appointment, a failover coordinator waits for its deadline.
+
+Note that a failover coordinator doesn't work with replica sets with two or more read-write instances.
+In this case, a coordinator logs a warning to stdout and doesn't perform any appointments.
+
+..  NOTE::
+
+    :ref:`anonymous replicas <configuration_reference_replication_anon>` are not considered candidates to be master.
+
+
+.. _supervised_failover_overview_perform_failover:
+
+Performing a failover
+~~~~~~~~~~~~~~~~~~~~~
+
+A failover coordinator monitors the statuses of instances by sending requests to instances each ``probe_interval`` seconds.
+While an appointment for a master instance exists and a read-write deadline has not expired, a failover coordinator does nothing.
+If all attempts to renew the deadline are failed during some time interval, a leader switches to read-only mode.
+
+..  NOTE::
+
+    If a remote etcd-based storage is used to maintain the state of failover coordinators, you can also perform a :ref:`manual failover <supervised_failover_manual>`.
+
+
+
+
+.. _supervised_failover_overview_fault_tolerance:
+
+Active and passive coordinators
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To increase fault tolerance, you can :ref:`run <supervised_failover_start_coordinator>` two or more failover coordinators.
+In this case, only one coordinator is active and used to control leadership in a replica set.
+Other coordinators are passive.
+Passive coordinators don't perform any read-write appointments and decline all the existing appointments.
```

Comment:
There is no process of declining an appointment from another coordinators. A coordinator may discard its own appointment as part of the manual switchover process. However, it seems, it is not what we say about here. Possibly a mistake.